shared/js/background/settings.es6.js

```diff
@@ -112,27 +111,6 @@ function logSettings () {
     })
 }
 
-function registerListeners(){
-    chrome.runtime.onMessage.addListener(onUpdateSetting);
-    chrome.runtime.onMessage.addListener(onGetSetting);
-}
-
-var onUpdateSetting = function(req, sender, res) {
-    if(req.updateSetting) {
-        var name = req.updateSetting['name'];
-        var value = req.updateSetting['value'];
-        updateSetting(name, value);
-    }
-};
-
-var onGetSetting = function(req, sender, res){
-    if(req.getSetting){
```

Comment:
I think you forgot to move this over to the consolidated messages? Whitelist a site then check the settings page.

Reply:
Good catch, it was there but I'd done a bungled copy + paste. Fixed now.